### PR TITLE
Pazricardo portfolio

### PR DESCRIPTION
**Pull request by pazricardo**
Change in appengine-web.xml to my project ID

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,15 +2,30 @@
...
+      <p>I am a First Generation Latino Student as well. I am a Mexican Honduran American.
+          I love my culture and try to show others why I am who I am. 
+      </p>
+      <p> LinkedIn Profile: www.linkedin.com/in/pazricar
+          GitHub Profile: in the process
+      </p>
+      <p> I am currently very involved on campus. I am the Secretary for the Largest Student 
+          organization on campus. It is called Culturas de las Razas Unidas. CRU for short.
+          I am also part of the Vice President and Chief Diversity Officer Search Committee for 
+          Michigan State University.
```
Comment (atreil) [inline portfolio/src/main/webapp/index.html]: I think you are missing a closing `</p>` tag.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -2,15 +2,30 @@
...
+          I love my culture and try to show others why I am who I am. 
+      </p>
+      <p> LinkedIn Profile: www.linkedin.com/in/pazricar
+          GitHub Profile: in the process
+      </p>
+      <p> I am currently very involved on campus. I am the Secretary for the Largest Student 
+          organization on campus. It is called Culturas de las Razas Unidas. CRU for short.
+          I am also part of the Vice President and Chief Diversity Officer Search Committee for 
+          Michigan State University. 
+      <img src="images/Luis7.JPG" style="width:500px;height:400px;/>
```
Comment (atreil) [inline portfolio/src/main/webapp/index.html]: I think you need to commit `images/Luis7.JPG` as well, otherwise I won't be able to see it :)

Comment (pazricardo) [reply portfolio/src/main/webapp/index.html]: I am gonna need some help with commiting, still kinda new to it.

Comment (atreil) [reply portfolio/src/main/webapp/index.html]: So assuming that you added your image according to [these instructions](https://github.com/pazricardo/software-product-sprint-or-my-portfolio/blob/master/walkthroughs/week-1-web-development/portfolio-walkthrough.md#uploading-images), then you should probably run:

`git add images/Luis7.JPG`
`git commit`

**Pull request merged**